### PR TITLE
Facia Press pressed json endpoints

### PR DESCRIPTION
**Pull request by janua**
It is handy to be able to see what `facia-press` will generate.

@robertberry @stephanfowler

### CODE REVIEW

File: facia-press/app/controllers/Application.scala [hunk 1]
```diff
@@ -1,14 +1,25 @@
 package controllers
 
+import frontpress.FrontPress
+import play.api.libs.json.Json
 import play.api.mvc.{Action, Controller}
 import services.ConfigAgent
 
 object Application extends Controller {
+  import scala.concurrent.ExecutionContext.Implicits.global
```
Comment (gklopper) [inline facia-press/app/controllers/Application.scala]: We tend to mixin the `ExecutionContexts` trait.

File: facia-press/app/controllers/Application.scala [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
   def index = Action {
     Ok("Hello, I am the Facia Press.")
   }
-  
+
   def showCurrentConfig = Action {
     Ok(ConfigAgent.contentsAsJsonString).withHeaders("Content-Type" -> "application/json")
```
Comment (gklopper) [inline facia-press/app/controllers/Application.scala]: These need Cache headers

File: facia-press/app/controllers/Application.scala [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
   }
+
+  def generateDraftPressedFor(path: String) = Action.async { request =>
```
Comment (gklopper) [inline facia-press/app/controllers/Application.scala]: Needs Auth if it lets draft escape...

Comment (gklopper) [inline facia-press/app/controllers/Application.scala]: Unless there is a pressing (excuse the pun) need to the draft endpoint just drop it. I we really need it come talk to me today.

**Comment by stephanfowler** [comment]
:+1: valuable for dev/debugging

**Pull request merged**